src/views/vmd-rdv.view.ts

```diff
@@ -281,6 +293,7 @@ export abstract class AbstractVmdRdvView extends LitElement {
                     this.lieuxParDepartementAffiches);
             } finally {
                 this.searchInProgress = false;
+                this.launchCheckingUpdates();
```

Comment:
@Aymerik je pense pas que ce soit une bonne idée de démarrer le check de maj ici, car je pense qu'on doit pouvoir appeler le `refreshLieu()` alors que le `setInterval()` est déjà en cours.

J'aurais plutôt : 
- Appelé le `launchCheckingUpdates()` uniquement lors du clic sur le bouton `Rafraîchir`
- Checké (histoire d'être sûr et certain) qu'on démarre le `setInterval()` uniquement si `this.lieuBackgroundRefreshIntervalId` est `undefined`